modules/service/src/main/scala/lucuma/odb/service/Services.scala

```diff
@@ -0,0 +1,202 @@
+   * instance. The purpose is as follows:
+   *
+   *   - When `Services` is `given` and `Services.Syntax.*` is imported, all members here are 
+   *     available unprefixed; i.e., you can just say `groupService.doWhatever(...)` or `user`
+   *     or `session` and the reference will be taken from the implicit `Services` in scope.
+   *   - When `Transaction` is `given` and `Services.Syntax.*` is imported, the current transaction
+   *     is available as `transaction`. This is rarely necessary, but it allows service code to use 
+   *     savepoints and rollback to handle errors when necessary.
+   *   - Most service methods have a `using Transaction[F]` clause, to prevent them from being
+   *     called without an active transaction. This adds a bit of safety.
+   * 
+   * See also `Services.Syntax.useTransactionally`, which is more commonly used; it does the same
+   * thing but works on `Resource[F, Service[F]]`. In this form you never actually see the
+   * `Services` instance; it's always implicit.
+   */
+  def transactionally[A](fa: (Transaction[F], Services[F]) ?=> F[A])(using NoTransaction[F]): F[A]
+
+  /** The `AllocationService`. */
+  def allocationService: AllocationService[F]
+  
+  /** The `AsterismService`. */
+  def asterismService: AsterismService[F]
+  
+  /** The `GeneratorParamsService`. */
+  def generatorParamsService: GeneratorParamsService[F]
+  
+  /** The `GmosLongSlitService`. */
+  def gmosLongSlitService: GmosLongSlitService[F]
+  
+  /** The `GroupService`. */
+  def groupService: GroupService[F]
+  
+  /** Construct an `ObsAttachmentFileService`, given an `S3FileService`.  */
+  def obsAttachmentFileService(s3: S3FileService[F]): ObsAttachmentFileService[F]
+  
+  /** The `ObsAttachmentMetadataService`. */
+  def obsAttachmentMetadataService: ObsAttachmentMetadataService[F]
+  
+  /** The `ObservationService`. */
+  def observationService: ObservationService[F]
+  
+  /** The `ObservingModeServices`. */
+  def observingModeServices: ObservingModeServices[F]
+  
+  /** The `PartnerSplitsService`. */
+  def partnerSplitsService: PartnerSplitsService[F]
+  
+  /** The `ProgramService`. */
+  def programService: ProgramService[F]
+  
+  /** Construct a `ProposalAttachmentFileService`, given an `S3FileService`. */
+  def proposalAttachmentFileService(s3: S3FileService[F]): ProposalAttachmentFileService[F]
+  
+  /** The `ProposalAttachmentMetadataService`. */
+  def proposalAttachmentMetadataService: ProposalAttachmentMetadataService[F]
+  
+  /** The `ProposalService`. */
+  def proposalService: ProposalService[F]
+  
+  /** The `SmartGcalService`. */
+  def smartGcalService: SmartGcalService[F]
+  
+  /** The `TargetService`. */
+  def targetService: TargetService[F]
+  
+  /** The `TimingWindowService`. */
+  def timingWindowService: TimingWindowService[F]
+
+  /** Construct an `Itc` service, given an `ItcClient`.*/
+  def itc(itcClient: ItcClient[F]): Itc[F]
```

Comment:
Right now we just use the one client, `ItcClient` but I wonder if we'll need something to manage clients or if they should be included as generally available like services.

Reply:
Yeah, I guess it depends on how it works on the back end. I think you only generally need a single http client in an app.